### DISCUSSION TITLE
How to add new Partner categories

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16860] [How to] [answered]
Hello
I would like to add a category to the partner.
Unfortunately, I did not find where can I add categories.
Can you please help me?

![image](https://user-images.githubusercontent.com/20862325/217038464-38c7351e-6ed9-4004-a2fd-10f418708a83.png)

**Comment by rskrzypczak** [comment, marked as answer]
You have it in the _Software configuration/Standard modules/Fields – Folders_ panel
![image](https://user-images.githubusercontent.com/10221999/217163015-f7b014d3-b1b5-4394-a479-0c0770d5d86f.png)
